test/parallel/test-process-replace-fail.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const { mustCall } = require('../common');
+const { throws } = require('assert');
+
+// Non existent path name
+{
+  throws(mustCall(() => {
+    process.replace(
+      process.execPath + '_non_existing',
+      [__filename, 'replaced'],
+      { EXECVE_A: 'FIRST', EXECVE_B: 'SECOND', CWD: process.cwd() }
```

Comment:
You will need to extend, rather than replace, the environment to fix the sharedlib builds.

Reply:
Thanks for the hint! I'll update the test to include `process.env`.